lib/OpenQA/Schema/Result/AuditLog.pm

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2015 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+package OpenQA::Schema::Result::AuditLog;
+use strict;
+use base qw/DBIx::Class::Core/;
+
+# use db_helpers;
+
+__PACKAGE__->table('audit_log');
```

Comment:
we use plural for all other result classes - and tables. I admit that logs sounds weired, but it's still inconsistent. Perhaps AuditLogEntries?